app/api/v1/views.py

```diff
@@ -0,0 +1,113 @@
+"""This is the view for sendit application"""
+from flask import Flask, jsonify
+from flask_restplus import Api, Resource
+from .models import UserModel
+from .models import ParcelModel
+
+
+app = Flask(__name__)
+api = Api(app)
+
+
+class Users(Resource):
+    """This is the user class for signup"""
+    def post(self):
+        """The post request for the class"""
+        payload = api.payload
+        # check if payload is empty
+        if not payload:
+            return {'status': 'failed', 'message': 'provide all fields'}, 400
+        if payload:
```

Comment:
You do not need this if statement since you've checked for empty payload above.

Reply:
Alright ill try and remove the repetitive  if statements